Orange/classification/elliptic_envelope.py

```diff
@@ -0,0 +1,27 @@
+import sklearn.covariance as skl_covariance
+
+from Orange.classification import SklLearner
+
+__all__ = ["EllipticEnvelopeLearner"]
+
+
+class EllipticEnvelopeLearner(SklLearner):
+    __wraps__ = skl_covariance.EllipticEnvelope
+    name = 'elliptic envelope'
+
+    def __init__(self, store_precision=True, assume_centered=False,
+                 support_fraction=None, contamination=0.1,
+                 random_state=None, preprocessors=None):
+        super().__init__(preprocessors=preprocessors)
+        self.params = vars()
+        self.clf = None
+
+    def fit(self, X, Y=None, W=None):
+        self.clf = self.__wraps__(**self.params)
+        if W is not None:
+            return self.__returns__(self.clf.fit(X, W.reshape(-1)))
+        return self.__returns__(self.clf.fit(X))
+
+    def mahalanobis(self, observations):
```

Comment:
If we wrap the original method so that the Orange learner also provides it,
we should probably allow observations to be an Orange.data.Table?